opencodelists/models.py

```diff
@@ -3,6 +3,43 @@
 from django.utils.text import slugify
 
 
+class UserManager(BaseUserManager):
+    """
+    Custom user model manager where email is the unique identifiers
+    for authentication instead of usernames.
+    """
+
+    def create_user(self, username, email, password, **extra_fields):
+        """Create and save a User with the given email and password."""
+        if not email:
+            raise ValueError("The Email must be set")
+
+        org = Organisation.objects.first()
```

Comment:
Maybe for later: could the organisation slug be passed in as a required argument?

Reply:
I had a quick look for this and couldn't find an easy way to make it happen with `createsuperuser`.

Adding a custom command might be quicker to get that.